src/main/java/md2k/mCerebrum/cStress/features/ECGFeatures.java

```diff
@@ -361,20 +363,16 @@ private void filterPeaksTemp2(DataPointStream Rpeak_temp2, DataPointStream Rpeak
 
         while (!difference) {
             int length_Rpeak_temp2 = Rpeak_temp2.data.size();
-            List<DataPoint> diffRpeak = new ArrayList<DataPoint>();
+            List<Long> diffRpeak = new ArrayList<Long>();
```

Comment:
Hi Tim.
I wasn't seeing any need for it.  It didn't have a datastream, and all we needed were the timestamp differences.  I also didn't see any need to subtract the values of the peaks.  If you're anticipating saving these into datastream, then of course, add them back into a DataPoint list.